cmd/log-splicer/main.go

```diff
@@ -204,23 +250,31 @@ func main() {
 		if !has {
 			continue
 		}
+		a.latest += 1
+		feeds[msg.author.Ref()] = a
 
 		_, err = a.log.Append(v)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "failed to write entry to output log %s: %s\n", logPaths[1], err)
+			fmt.Fprintf(os.Stderr, "failed to write entry to output log %s: %s\n", outdir, err)
```

Comment:
meta comment: If this `fmt.Fprint(os.Stderr, ...); os.Exit(1)` style becomes to repetetiv, you could try something like this [here](https://github.com/cryptoscope/ssb/blob/1c7184ced3f9378528cc22fcf7f0e327bfd2251a/cmd/go-sbot/main.go#L397):

```go
func spliceLogs() error {
  // actual work with "normal" error handling
  // ...
}

func main() {
  //  flags stuff
  flag.Parse()
  
  err:= sliceLogs()
  if err!=nil {
    fmt.Fprintf(os.Stderr, ... )
    os.Exit(1)
  }
}
```

Reply:
ahh nice, so extract all the stuff i am doing in `main` to an encapsulating routine (e.g. `spliceLogs)`? that's a really good point, i was actually thinking that things were getting repetitive

Reply:
yes. And since this is a single-use CLI tool, you even get away with globals for the flag variables.

(In contrast to an rpc like thing where these request params would need to be passed)